### ISSUE TITLE
[GA4] Convert current GA API queries to use the Analytics Data API for GA4

### ISSUE THREAD

**Issue by escattone** [mozilla/sumo#1705]
We need to convert the code in https://github.com/mozilla/kitsune/blob/main/kitsune/sumo/googleanalytics.py to use the [Analytics Data API for GA4](https://developers.google.com/analytics/devguides/reporting/data/v1).

## Data Engineering Help
We currently have two GCP service accounts defined -- one in our `data-sumo-nonprod` project and another in our `data-sumo-prod` GCP project -- that our server-side code uses to access GA3 data via the old `Google Analytics API`, and we'd like to update that server-side code to use the new GA4 `Google Analytics Reporting API`.  My initial understanding is that we'd need the data engineering team to do the following:

- [x] Enable the GA4 `Google Analytics Reporting API` in both of the `data-sumo-nonprod` and `data-sumo-prod` GCP projects
- [x] Give our stage service-account read-only access to our stage GA4 property
- [x] Give our production service-account read-only access to our production GA4 property

**Comment by escattone** [comment]
https://mozilla-hub.atlassian.net/browse/DO-1603

**Comment by escattone** [comment]
All of the tasks that we needed done by the data engineering teams have been completed.

**Comment by escattone** [comment]
I found an issue during QA. The issue is that I'm not counting page views properly for a localized URL that's using the same slug as its English parent, and the localized article doesn't exist. In that case, we display the English article at the non-English locale, so the counts should be attributed to the English article. The way the code is right now, the counts are not attributed to any article.

I created https://github.com/mozilla/sumo/issues/1768 to track this.
